src/style.css

```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 100px;
+  color: rgb(255, 255, 255);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 100%;
+  flex-grow: 1;
```

Comment:
after you have set `flex-basis`, `flex-grow` does not do anything

Reply:
the only thing it does here is it saves you from the issue in my last comment. but it can be avoided if you set the `flex-basis` properly there

Reply:
do not `flex-grow` anymore